docs/guides/app-distribution.md

```diff
@@ -0,0 +1,191 @@
+---
+layout: default
+title: "App Distribution (OAuth)"
+lang: en
+---
+
+# App Distribution (OAuth)
+
+A newly created Slack app can only be installed in its development workspace in the beginning. By setting an OAuth Redirect URL and enabling [App Distribution]((https://api.slack.com/start/distributing)), the app becomes to be ready for installation in any other workspaces.
+
+* [Using OAuth 2.0](https://api.slack.com/docs/oauth)
+* [Distributing Slack Apps](https://api.slack.com/start/distributing)
+
+### Slack App Configuration
+
+To enable App Distribution, visit the [Slack App configuration page](http://api.slack.com/apps), choose the app you're working on, go to **Settings** > **Manage Distribution** on the left pane, and follow the instructions there. For **Redirect URL**, Bolt apps respond to `https://{your app's public URL domain}/slack/oauth/callback` if you go with recommended settings.
```

Comment:
"if you go with recommended settings."

What are the configuration options if the developer doesn't want to use the default settings?